src/internal/sqs.ts

```diff
@@ -64,6 +64,22 @@ export class SQSClient extends AWSClient {
             body = { ...body, MessageGroupId: options.messageGroupId }
         }
 
+        if (typeof options.messageAttributes !== 'undefined') {
+            const attributeParameters = Object.entries(options.messageAttributes).reduce((params, [name, attribute], i) => {
```

Comment:
This specific code would deserve some comment explaining what's going on. I had to read the docs and execute and log the code to figure it out myself: based on the attributes type, we match it to a specific object shape with the value field being different for each 😄 🙇🏻

Reply:
Done!

Reply:
@nickcaballero It worked now. Thanks for your help.

Reply:
Merci 🙇🏻